content/docs/user-guide/deploying/index.md

```diff
@@ -22,17 +22,16 @@ configuration. You just need your model saved with MLEM and an environment you
 want to deploy to
 
 ```yaml
-$ mlem deployment run <env type> <name> --model <model name> --some_option
+$ mlem deployment run <env type> <name> \ --model <model name> \ --some_option
```

Comment:
This doesn't render as a linebreak if that's what you aimed for... if not I'm confused :)

Reply:
It seems to be missing the actual line breaks:
```suggestion
$ mlem deployment run <env type> <name> \
                      --model <model name> \
                      --some_option
```